### DISCUSSION TITLE
How to delete empty folders after import?

### DISCUSSION THREAD

**Discussion by StefanDorschu** [beetbox/beets#4852] [Q&A] [answered]
When i import using the "move" option, beets leaves behind a lot of empty folders. Is there a way to clean that up using beets? Or is there a useful script to do so?

Thank for your help,

Stefan

**Comment by jackwilsdon** [comment]
Are you on Windows or Linux? 

For Windows, you could try one of the answers from this Super User question: https://superuser.com/questions/39674/recursively-delete-empty-directories-in-windows

For Linux, you could try `find /path/to/your/music/folder -type d -empty -delete` which will delete empty folders.

To run these automatically, take a look at the `hook` plugin: https://docs.beets.io/en/latest/plugins/hook.html

**Comment by StefanDorschu** [comment]
Thank you for your reply!

**Comment by sampsyo** [comment, marked as answer]
Oddly, the importer is actually supposed to do this automatically, out of the box. The `clutter` config option has a little more detail:
https://beets.readthedocs.io/en/stable/reference/config.html#clutter

Namely, non-root directories that seem to be empty should be pruned. If you type `beet import /path/to/dir`, `/path/to/dir` itself will never be removed, but empty descendants of that should be deleted. If they're not, there could be invisible files in there that are preventing beets from considering them "empty."

**Reply by StefanDorschu** [reply]
You are right, thank you :)
